wp-job-manager.php

```diff
@@ -112,6 +113,9 @@ public function __construct() {
 		add_action( 'wp_logout', array( $this, 'cleanup_job_posting_cookies' ) );
 		add_action( 'init', array( 'WP_Job_Manager_Email_Notifications', 'init' ) );
 
+		// Filters
+		add_filter( 'wp_privacy_personal_data_exporters', array( 'WP_Job_Manager_User_Data_Exporter', 'register_wpjm_user_data_exporter' ) );
```

Comment:
This class is called `WP_Job_Manager_Data_Exporter`. As mentioned earlier, this method also needs to be a static function when using this type of callable reference.